msgservice/logs.go

```diff
@@ -3,12 +3,11 @@ package msgservice
 import "log"
 
 func prnInputLogs(msg string, v ...interface{}) {
+	maxLen := 20
```

Comment:
why do use a magic number - 20?

Reply:
the goal of entire function is to reduce amount of logging. If one of parameters passed is byte array then it's size is limited to 20 bytes, other parameters are go as is. I believe similar logic existed before:
https://github.com/aquasecurity/postee/commit/770ba7ef2ecf3d9ad9cd17f9619365fdbc75e1b7
(see changes in `logs.go`)